lib/grovepi/pivotpi.ex

```diff
@@ -8,6 +8,8 @@ defmodule GrovePi.PivotPi do
   channel 1. You must initialize the PivotPi board using `PivotPi.start()`.
 
   ```elixir
+  iex> alias GrovePi.PivotPi
```

Comment:
It is suggested to not alias modules in documentation. It allows a copy and paste withought having to go up to the alias and copy that.

Reply:
ah ok!  I'll fix for both modules.